ykrt/src/compile/jitc_yk/codegen/x64/lsregalloc.rs

```diff
@@ -1807,6 +1830,9 @@ pub(crate) enum GPConstraint {
         force_reg: Option<Rq>,
         can_be_same_as_input: bool,
     },
+    /// Align `op`'s upper bits to `out_ext`: the instruction must not use the resulting register
```

Comment:
I'm not sure what it means to align an operand's upper bits. Can you elaborate?

Reply:
"Align extensions" is the terminology we use elsewhere. So this should be "Align op's extension to out_ext" or similar if that's OK?

Reply:
I've just read the implementation of `align_extensions()` and now see the confusion.

Here "align" isn't about memory alignment, it's about **satisfying** extension requirements.

I think naming like `unify_extensions()` would make more sense, but that's one for another day.

I'm happy to go with whatever you want in light of this epiphany :)

Reply:
Yes, another term might be a good idea for a future PR.